variant-service/api/restapi/configure_variant_service.go

```diff
@@ -110,10 +120,35 @@ func configureAPI(api *operations.VariantServiceAPI) http.Handler {
 			return operations.NewMainGetVariantsInternalServerError().WithPayload(errPayload)
 		}
 
-		query := tx.Where(fmt.Sprintf("chromosome = '%s' AND start BETWEEN %d AND %d",
-			*params.Chromosome, *params.Start, *params.End))
+		conditions := ""
+
+		if params.Chromosome != nil {
+			conditions = fmt.Sprintf(addAND(conditions) + "chromosome = '%s'", *params.Chromosome)
+		} else {
+			// TODO do we want to do this? If so, add 422001 to error code doc. If not, remove here and in swagger.yml
```

Comment:
I don't think we need to do this; if the query isn't clearly invalid or likely to be harmful, I don't think we need to write code to forbid any particular search combination.